docker/entrypoint.sh

```diff
@@ -75,13 +75,15 @@ case ${entry_cmd} in
 	    case ${test_py} in
 	        tests/test_elasticsearch__provider.py)
 	        ;&
+	        tests/test_sensorthings_provider.py)
+	        ;&
 	        tests/test_postgresql_provider.py)
 			;&
 	        tests/test_mongo_provider.py)
 	        	echo "Skipping: ${test_py}"
 	        ;;
 	        *)
-	        	python3 -m pytest ${test_py}
+	        	pdm run python3 -m pytest ${test_py}
```

Comment:
Do you know of a way to signal test failures to the caller, mainly the GitHub Workflow in containers.yml? Ultimately we want to run these tests in that Workflow, and not push the Docker Image on test failures. This issue is already existing. Or mainly return and catch an error exit code?

Reply:
Sure. We can run the image without pushing it, running the test, and avoid pushing it if there is a failure.